index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="vewport" content="width=device-width, initial-scale=1.0">
+    <title>Documents</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+ <boby>
+    <nav class="navbar">
+        <div class="name">Patrick & Amos</div>
+        <div class="links">
+            <a href="#about">About</a>
+            <a href="#project">Project</a>
+            <a href="#contact">Contact</a>
+        </div>
+    </nav>
+
+    <div class="main-section">
+        <section id="about">
+            <header class="header">
+                <h1>About</h1>
+                <h2>Patrick & Amos</h2>
+                <h2>Web Developer</h2>
+            </header>
+            <div class="aboutme">
+                <p>
+                    we are skilled web developer that enjoys being practical in our projects
+                </p>     
+            </div>
+                <section id="projects">
+                    <header class="header">
+                        <h1>Projects</h1>
+                        <h2>Here are some of the projects we made</h2>
+                    </header>
+                    <DIV class="project">
```

Comment:
Hey, Great job! so far.
I think that it would be nice to use lowercase for elements and attribute names().